app/models/user.rb

```diff
@@ -1,4 +1,40 @@
 class User < ApplicationRecord
-  has_many :book_rooms, :guests, :rooms, :reports, dependent: :destroy
-  belongs_to :role, :group
+  VALID_EMAIL_REGEX = Settings.model.user.email_validate_regex
+  USER_PARAMS = Settings.user_params
+
+  has_many :book_rooms, dependent: :destroy
+  has_many :guests, dependent: :destroy
+  has_many :rooms, dependent: :destroy
+  has_many :reports, dependent: :destroy
+  belongs_to :role
+  belongs_to :group
+
+  validates :name, presence: true
+  validates :email, presence: true, length: {maximum: Settings.model.user.email_max_length},
```

Comment:
đoạn này bị quá 80 ký tự à, cho nó xuống dòng đi
```suggestion
  validates :email, presence: true, 
    length: {maximum: Settings.model.user.email_max_length},
```